system-test/spanner.ts

```diff
@@ -3217,9 +3231,9 @@ describe('Spanner', () => {
             assert.ifError(err);
 
             // We just want the two most recent ones.
-            rows.splice(0, rows.length - 2);
+            rows!.splice(0, rows!.length - 2);
 
-            rows = rows.map(x => x.toJSON());
+            rows = rows!.map(x => x.toJSON()) as Row[];
```

Comment:
I think this should be [`as Json[]`](https://github.com/googleapis/nodejs-spanner/blob/master/src/codec.ts#L32)

Reply:
`rows` is type of `Row[]` while using `Json[]` it gives below error:
`Type 'Json[]' is not assignable to type 'Row[]'.`

Reply:
I see. I'm guessing it is because we're attempting to re-use the variable `rows` that is set to type `Row[]`. We may just want to assign it to a new variable altogether of type `Json[]`.